### PR TITLE
Added a footer and MLH Code of Conduct

### PR DESCRIPTION
**Pull request by sarj21**
This can be moved or restyled but we need it so

### CODE REVIEW

File: src/components/FooterSection.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name: 'FooterSection',
+};
+</script>
+
+<style lang="scss" scoped>
+  @import '../styles/vars';
+
+  #footer-section {
+    width: 100%;
+    min-height: 100px;
```
Comment (mecolmg) [inline src/components/FooterSection.vue]: I don't think we need a min height for the footer

File: src/components/FooterSection.vue [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    padding: 20px;
+    align-self: center;
+  }
+
+  a {
+    font-family: $font-VT232;
+    color: yellow;
+    background-color: blue;
+  }
+
```
Comment (mecolmg) [inline src/components/FooterSection.vue]: Remove extra spacing

File: src/components/FooterSection.vue [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  .code-of-conduct {
+    padding: 20px;
+    align-self: center;
+  }
+
+  a {
+    font-family: $font-VT232;
+    color: yellow;
+    background-color: blue;
```
[the remark this answers is not in the document]
Comment (sarj21) [reply src/components/FooterSection.vue]: Yeah i just went with what default i had as i figured the footer would change

File: src/components/FooterSection.vue (hunk 1 above)
Comment (sarj21) [reply src/components/FooterSection.vue]: I set one because i figured it might have more stuff for MLH, not sure

**Comment by sarj21** [comment]
I added it to the homepage on my end but didnt push it since i knew it would change

**Pull request merged**